bulk_indexer_test.go

```diff
@@ -165,3 +165,36 @@ func TestDynamicTemplates(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, int64(2), stat.Indexed)
 }
+
+func TestPipeline(t *testing.T) {
+	client := docappendertest.NewMockElasticsearchClient(t, func(w http.ResponseWriter, r *http.Request) {
```

Comment:
would be good to assert that pipeline is successfully passed in the request

Reply:
Updated the test so it checks that the pipelines made it.